### PR TITLE
fix some typo

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 ## Overview
 
-This project aims to provide a simple and complete user interface for your private docker registry. You can customize the interface with various options. The major option is `SINGLE_REGISTRY` which allows you to disable the dynamic selection of docker registeries (same behavior as the old **static** tag).
+This project aims to provide a simple and complete user interface for your private docker registry. You can customize the interface with various options. The major option is `SINGLE_REGISTRY` which allows you to disable the dynamic selection of docker registers (same behavior as the old **static** tag).
```
Comment (Joxit) [inline README.md:11]: Revert this line, registryies is correct

File: dist/docker-registry-ui.js [hunk 2]
```diff
@@ -37,5 +37,5 @@ function dt(e,t){(function(e){return it(e,n)||e.tagName.toLowerCase()})(e)!==t&&
...
...:"on-server-change",evaluate:e=>e.props.onServerChange}],redundantAttribute:"expr67",selector:"[expr67]"},{type:r.IF,evaluate:e=>!e.props.readOnlyRegistries,redundantAttribute:"expr68",selector:"[expr68]",template:e(null,[{type:r.TAG,getComponent:a,evaluate:e=>"remove-registry-url",slots:[],attributes:[{type:t.ATTRIBUTE,name:"opened",evaluate:e=>e.state["remove-registry-url"]},{type:t.EVENT,name:"on-close",evaluate:e=>e.onClose("remove-registry-url")},{type:t.EVENT,name:"on-notify",evaluate:e=>e.props.onNotify},{type:t.EVENT,name:"on-server-change",evaluate:e=>e.props.onServerChange}]}])},{type:r.TAG,getComponent:a,evaluate:e=>"material-button",slots:[{id:"default",html:'<i class="material-icons">more_vert</i>',bindings:[]}],attributes:[{type:t.EVENT,name:"onClick",evaluate:e=>e.onClick}],redundantAttribute:"expr69",selector:"[expr69]"},{type:r.TAG,getComponent:a,evaluate:e=>"material-dropdown",slots:[],attributes:[{type:t.ATTRIBUTE,name:"items",evaluate:e=>e.dropdownItems.filter((t=>t.ro||!e.props.readOnlyRegistries))},{type:t.EVENT,name:"on-click",evaluate:e=>e.onDropdownSelect},{type:t.ATTRIBUTE,name:"opened",evaluate:e=>e.state.isDropdownOpened}],redundantAttribute:"expr70",sel
```
Comment (Joxit) [inline dist/docker-registry-ui.js:40]: Revert this change

File: src/components/version-notification.riot [hunk 3]
```diff
@@ -3,7 +3,7 @@
   <material-popup opened="{ state.open }" onClick="{ onClose }">
     <div class="material-popup-title">Check for updates</div>
     <div class="material-popup-content">
-      <p>The version <b>{ state.tag_name }</b> of Docker Regisrty UI now available.</p>
+      <p>The version <b>{ state.tag_name }</b> of Docker registry UI now available.</p>
```
Comment (Joxit) [inline src/components/version-notification.riot:6]: This is the name of project, keep the upercase, revert this change

**Review by Joxit — CHANGES_REQUESTED** [review]
Hi, thank you for using my project and submitting PR. I made some comments, could you update your PR ?

File: src/components/version-notification.riot (hunk 3 above)
Comment (cuishuang) [reply src/components/version-notification.riot]: Thanks. Done

File: src/components/version-notification.riot [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <material-popup opened="{ state.open }" onClick="{ onClose }">
     <div class="material-popup-title">Check for updates</div>
     <div class="material-popup-content">
-      <p>The version <b>{ state.tag_name }</b> of Docker Regisrty UI now available.</p>
+      <p>The version <b>{ state.tag_name }</b> of Docker registy UI now available.</p>
```
Comment (Joxit) [inline src/components/version-notification.riot:6]: Typo here, I ment keep the uppercase and fix the typo => `Docker Registry UI`

**Review by Joxit — CHANGES_REQUESTED** [review]
Last change before merge

File: src/components/version-notification.riot (hunk 4 above)
Comment (cuishuang) [reply src/components/version-notification.riot]: Thanks.  Done

**Review by Joxit — APPROVED** [review]

**Comment by Joxit** [comment]
Thanks for your contribution 😊
